app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
If it's possible to improve the menu somehow, maybe by adding borders/hovers or something to add some style. See https://docs.garden.io/using-garden as example.